config/config/src/Config.ts

```diff
@@ -214,6 +214,7 @@ export interface Config {
   strictStorePkgContentCheck: boolean
   managePackageManagerVersions: boolean
   strictDepBuilds: boolean
+  syncInjectedDepsAfterScripts?: boolean | string[]
```

Comment:
I don't think boolean should be supported. It doesn't make sense to sync after any script run.

Reply:
https://github.com/pnpm/pnpm/pull/9100/commits/ba40bbc9a63b60796d205e3c1cce252c9127bbcf